README.md

```diff
@@ -48,16 +48,19 @@ To ease maintenance, the notebooks are made up of reusable components. To see wh
 Component name | Open in Colab | Tasks
 -- | -- | --
 [Environment](https://github.com/open-contracting/notebooks-ocds/blob/main/component_environment.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_environment.ipynb) | Install requirements, import packages, load extensions and configure the notebook.
-[Charts setup](https://github.com/open-contracting/notebooks-ocds/blob/main/component_charts.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_charts.ipynb) | Install charts requirements, import charts packages and define plot functions.
+[Cardinal setup](https://github.com/open-contracting/notebooks-ocds/blob/main/component_charts.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/open-contracting/notebooks-ocds/blob/main/component_charts.ipynb) | Install cardinal requirements, define coverage functions and calculate the field list for a given file_name.
```

Comment:
URLs are incorrect (charts instead of cardinal). Capitalize Cardinal. Not sure why `file_name` is underscored, as in the readme we don't refer to any `file_name`.

Reply:
Done